src/core/utils/props.ts

```diff
@@ -9,6 +9,17 @@ export type ClassNameProps = {
   className?: string;
 };
 
+export type StylingProps = {
+  /**
+   * Custom CSS style properties.
+   */
+  style?: React.CSSProperties;
+  /**
+   * Custom CSS class name.
+   */
+  className?: string;
```

Comment:
Just extend `ClassNameProps` here instead of copying same definition..

Reply:
You would need to change the places where ClassNameProps is used too, if you want to just remove that :)

Reply:
@maxime4000 are you planning to finish up this PR or you just want us to add the props you need for this component?